library/src/main/java/net/yslibrary/licenseadapter/internal/HeaderViewHolder.java

```diff
@@ -57,13 +50,11 @@ public void bind(HeaderWrapper item) {
     author.setText(entry.author());
 
     String type = entry.license() != null ? entry.license().name : "";
-    int typeVisibility = View.VISIBLE;
-    if (type == null || type.equals("")) {
-      typeVisibility = View.GONE;
-    }
-    licenseType.setVisibility(typeVisibility);
+    licenseType.setVisibility(TextUtils.isEmpty(type) ? View.GONE : View.VISIBLE);
     licenseType.setText(type);
 
-    link.setText(entry.link());
+    String url = entry.license().url;
```

Comment:
I think library's url and license's url(oh, I should've name both as `url`) are different.
I want to show library's url, but with this change LicenseAdapter seems to display license's url.(such as https://raw.githubusercontent.com/xxx/bbb.../LICENSE.txt)